h_transport_materials/property_database/steel_316L/steel_316L.py

```diff
@@ -246,11 +255,84 @@
 
 serra_solubility = Solubility(
     data_T=1 / serra_data["solx"] * htm.ureg.K,
-    data_y=serra_data["soly"] * htm.ureg.mol * htm.ureg.m**-3 * htm.ureg.Pa**-0.5,
+    data_y=serra_data["soly"] * SOLUBILITY_UNIT_MOL_VOLUME,
```

Comment:
`SOLUBILITY_UNIT_MOL_VOLUME `isn't specific enough. I understand this refactors the code a little bit, but for readability, I'd rather keep the full units expression here.

```suggestion
    data_y=serra_data["soly"] * htm.ureg.mol * htm.ureg.m**-3 * htm.ureg.Pa**-0.5,
```